README.md

```diff
@@ -148,7 +148,7 @@ Options:
 UnCSS can be used as a [PostCSS](https://github.com/postcss/postcss) Plugin.
 
 ```js
-postcss([ require('postcss-uncss').postcssPlugin ]);
+postcss([ require('postcss-uncss') ]);
```

Comment:
Hey @flip-it , I agree with @RyanZim. Do you think you can update this README to point at uncss as suggested, instead of postcss-uncss?